src/bin/zmmailboxdctl

```diff
@@ -80,24 +80,18 @@ case "$1" in
     fi
 
     echo -n "Starting mailboxd..."
+
     # shellcheck disable=SC2086
     /opt/zextras/common/bin/java \
-      -Dfile.encoding=UTF-8 \
-      $mailboxd_java_options \
+                -Dfile.encoding=UTF-8 \
```

Comment:
do an shfmt -i 2 -ci -w pass for indentation

Reply:
Done!